app/src/main/res/layout-land/activity_tendays.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:orientation="horizontal" android:layout_width="match_parent"
+              android:layout_height="match_parent">
+
+    <android.support.v7.widget.RecyclerView
```

Comment:
150dp seems too small for the text your showing, doesn't it? Maybe make it a little big bigger? 200-250?